### PR TITLE
Primary: Added Ethereal Lore Secondary: Added a Naming Methodology for Moths

### PR DESCRIPTION
**Pull request by challade**
Hello loreslaves, Challade here. I recently took up the mantle of creating lore for the Ethereals since they have none to begin with and were "lore starving". 

They're essentially Neurons for a gargantuan living slime floating in space the size of a planet. Their purpose is to experience life and then die to send up their experiences to the planet. The planet provides NT with some of its tech, slimes are part of its payment plan. 

I provided a nod to the great lizard human war that I've been tasked with writing, and I provided the basis for a mining pod prototype that I've been hearing from the Supply nerds. I also wrote the entire thing with a story so that people can remember it more readily. 

Most importantly: I hope you enjoy it. 🙂

### CODE REVIEW

**Comment by EOBGames** [comment]
I have no idea how you've managed it but you have a merge conflict.

**Comment by challade** [comment]
I’m sorry, what?

**Comment by challade** [comment]
I found it out, I think because I don't have write access I can't modify existing MD's. I don't really know. If there's any more issues, please let them be known.

File: The Moths.md [hunk 1]
```diff
@@ -82,8 +84,12 @@ The CEO of Nanotrasen had this to say in a statement:
 
 * Moths tend to be very deceptive with their language. If they have something negative to say to someone they will often not say it at all, this leads to resentment among their peers.
 
+
+* Moths often form monogamous relationships and choose their mate primarily by smell rather than looks. Their powerful sense of smell can penetrate the masking of most deodorants.
+
 * Moths often form monogamous relationships and choose their mate primarily by smell rather than looks.
```
Comment (optimumtact) [inline The Moths.md:90]: this is the same line

File: The Moths.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ Part of this phenomenon is their seamless assimilation to their mostly human nei
 
 How it is that moths came to forget their own home planet is beyond comprehension. At most, some of the more elderly ones say they just saw a station or a colony and decided to just "hang out."
 
+Common interactions between moths and humans often involve clothes related scandals as moths often crave clothing for easy Keratine to aid in the maintenance and sheen of their scales. The issue comes when they prowl for worn items because they carry with them a "scent" or as they like to call it, a flavor. 
+=======
```
Comment (optimumtact) [inline The Moths.md:36]: this is not how you resolve conflicts

**Comment by optimumtact** [comment]
There's far too much detail in the backstory here

>This is not an exhaustive effort to catalog the universe of SS13 in great detail, as with Baystation. Rather, it is a collection of ideas and starting points for you to use IC with other players to generate interesting conversations and events. It is as close to canonical lore as you will get for the /tg/station codebase, however people are not required to use this lore, nor are maintainers bound to it.

We're not writing stories, we're giving cliffnotes so people can write their own story

**Comment by challade** [comment]
Sorry, I don't use github, on my end it didn't double, don't know what the issue is here, if you could give me some pointers on how to fix it, that'd be nice.

The Ethereal players I presented it to were receptive of it, it wraps up the entire race quite nicely and I incorporated some ideas that the players wanted in. Most importantly, I didn't **force** any roleplay mechanics that aren't already in the race gameplay.

The story isn't that long all things considered. It can be read in less than ten minutes, it's optional, and the only thing that is really at most necessary to read is the short blurb on the bottom about the naming procedures.
I mostly wrote the story to build a background for Ethereals, seeing as they have none, that's a completely optional read. 
My story is also meant to entertain, maybe inspire, any player that bothers reading it and want to write their own story. After all, you can't write a story about Spanish Conquistadors without Spain.

**Comment by challade** [comment]
Sure, I'll toss my pearls somewhere else then, it's your loss and the loss of the players that wanted this to go through.

**Pull request closed without merging**